projects/vdk-core/src/vdk/internal/builtin_plugins/config/log_config.py

```diff
@@ -57,12 +68,36 @@ def configure_loggers(
         "stream": "ext://sys.stderr",
     }
 
+    syslog_url, syslog_port, syslog_sock_type, syslog_enabled = syslog_args
+
+    if syslog_sock_type not in SYSLOG_SOCK_TYPE_VALUES_DICT.keys():
```

Comment:
Not related to this review. But this makes me think we should add support for enums in ConfigurationProvider - and this way it would automatically do this check.